src/main/java/Deadline.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+
+public class Deadline extends Task {
```

Comment:
Hello there! I think it'll be good to write a simple JavaDoc for this class so it will be easier for you to understand what this class does!